src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -2205,5 +2047,53 @@ public static String renderSanitizedFuncotationForVcf(final Funcotation funcotat
                 .map(field -> FuncotatorUtils.sanitizeFuncotationFieldForVcf(funcotation.getField(field)))
                 .collect(Collectors.joining(VcfOutputRenderer.FIELD_DELIMITER));
     }
+
+    /**
+     * Simple container class to represent bases that have been corrected for strandedness already.
+     * That is, if they would occur on {@link Strand#NEGATIVE}, they have been reverse-complemented.
+     * If they would occur on {@link Strand#POSITIVE}, they have not been altered.
+     */
+    public static class StrandCorrectedReferenceBases {
```

Comment:
Might want to add the Strand as a field here, so that the strand correction could be reversed if needed.

Reply:
Sure.

Fixed.